src/lib/mixer/mixer_group.cpp

```diff
@@ -198,6 +198,25 @@ MixerGroup::set_airmode(Airmode airmode)
 	}
 }
 
+unsigned
+MixerGroup::get_multirotor_count()
+{
+	Mixer	*mixer = _first;
+	unsigned rotor_count = 0;
+
+	while (mixer != nullptr) {
+
+		rotor_count = mixer->get_multirotor_count();
+
+		if (rotor_count > 0) {
+			break;
+
+		} else { mixer = mixer -> _next;}
```

Comment:
Replace with:
```
		}
                mixer = mixer->_next;
```

Reply:
Uh, In the original PR was like that but in the review I was suggested to use break for exiting the while loop. 
What is the best practice for exiting the while loop?
[The reference](https://en.cppreference.com/w/cpp/language/while) says that breaks can be used for exiting while loops, so i guess is just a madder of best choice?

Reply:
It's not about the break, you can leave it in. But if you `break` you don't need the `else`, and removing it makes the code more readable.
As for best practices, it depends: exiting a larger loop from within many nested cases is prone to errors and harder to understand. Adding a break in a simple loop on the other hand can make the code simpler.